resources/lib/ui/http.py

```diff
@@ -31,6 +32,11 @@ def Session():
         })
         _SESSION = s
 
+        token = TokenDecoder.get_token(lambda x: __send_request(s, x, None, None, False))
```

Comment:
Session shouldn't be aware of cookie not token.
the cookie should be attached to a specific request..
(just like i do with set referer)